app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -183,7 +188,7 @@ export default function CompetingStep({
 
   const handleEventSelection = ({ type, eventId }) => {
```

Comment:
This method is a bit inconsistent now. It's enforcing the max events only for `select_all_events` but not for `toggle_event`.

Reply:
true, but that is prevented in the component